chapter_09_docker.asciidoc

```diff
@@ -1348,6 +1348,9 @@ OK
 
 But we don't actually want to package up our database _inside_ the image, do we?
 We want the database on the server to have totally separate data from the one on our machine.
+// CSANAD: we need to list `src/db.sqlite3` in the .dockerignore file to achieve this.
+//         Otherwise, if the reader did not delete the DB, it would still end up built into
+// the image since the COPY directive copies everything that's inside `src`.
```

Comment:
YES!  thanks Csanad.   I was thinking of doing an explict `RUN rm db.sqlite3` in the dockerfile but this will be better, or at least equivalent.